xclim/temperature.py

```diff
@@ -75,33 +75,114 @@ def cfprobe(self, dan, dax):
                          compute=_ind.heat_wave_index,
                          )
 
-tmmean = Tas(identifier='tmmean',
-             units='K',
-             long_name="Mean daily mean temperature",
-             standard_name="air_temperature",
-             description="{freq} mean of daily mean temperature.",
-             keywords='',
-             compute=_ind.tg_mean, )
-
-tx_max = Tasmax(identifier='tx_max',
-                required_units='K',
-                long_name='Maximum temperature',
-                standard_name='tasmax',
-                description='Maximum daily maximum temperature.',
+tm_mean = Tas(identifier='tmmean',
+              long_name="Mean daily mean temperature",
+              units='K',
+              standard_name="air_temperature",
+              cell_methods='time: mean within days time: mean over days',
+              description="{freq} mean of daily mean temperature.",
+              keywords='',
+              compute=_ind.tg_mean, )
```

Comment:
I think it's confusing that the instance, the identifier and the compute function all have different names. We probably want to fix that before the next release.

Reply:
Yes.  I prefer the underscore but was trying at least to match variable 'id' with the cf-metadata spreadsheet from Lars